source/AuthManager.ts

```diff
@@ -0,0 +1,119 @@
+import 'reflect-metadata';
+import { ICognitoUserPoolLocator } from './ICognitoUserPoolLocator';
+import * as AWS from 'aws-sdk';
+import { ICognitoUserPoolApiModel } from './ICognitoUserPoolApiModel';
+import { AuthenticationDetails, CognitoUser, CognitoUserPool, CognitoUserSession } from 'amazon-cognito-identity-js';
+import { inject, injectable } from 'inversify';
+import TYPES from '../../ioc.types';
+import { Logger } from 'winston';
+
+// tslint:disable-next-line:no-string-literal no-var-requires
+global['fetch'] = require('node-fetch');
```

Comment:
This should be removed and left to the consumer. Browser consumers shouldn't need it at all, and NodeJS consumers should be free to choose whatever polyfill they would like to use.